charts/argocd-apps/Chart.yaml

```diff
@@ -3,4 +3,4 @@ name: argocd-apps
 description: An argocd app to deploy apps inside the virtual cluster
 type: application
 
-version: 0.3.2
+version: 0.3.3
```

Comment:
This should be minor as it adds a whole app

Reply:
changed to 0.4.0